composer.json

```diff
@@ -16,12 +16,17 @@
         "irc": "irc://irc.freenode.net/yii",
         "source": "https://github.com/yiisoft/event-dispatcher"
     },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
         "php": "^7.4",
         "psr/event-dispatcher": "1.0.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.0"
+        "infection/infection": "^0.16.3",
+        "phan/phan": "^3.0.2",
+        "phpunit/phpunit": "^9.2",
+        "yiisoft/di": "^3.0@dev"
```

Comment:
If we want to have EventDispatcherProvider, we have to more this dependency to `require`.

Reply:
done.